exercises/practice/automated-readability-index/.docs/instructions.md

```diff
@@ -0,0 +1,112 @@
+# Instructions
+
+As we grow up, we can understand more and more complex texts. But how is the level of complexity of a given text assessed, and how can we make programs which do that? 
+In this exercise, you will write a program which will determine how difficult a text is to read
+and what age it is best suited for.
+
+We will use the
+[Automated readability index (ARI)][wiki] formula for calculating the text complexity.
+The ARI was developed in 1967 and assesses the U.S. grade level required to read 
+a piece of text. In some ways, it is similar to other formulas. Its main difference 
+is rather than counting syllables, it counts characters. The more characters per word, 
+the harder the word. It also counts the number of words per sentences. This sets it apart from some 
+other formulas.
+
+## Text Preview
+
+Your program should display the first three sentences of the text, but no more than 50 characters. If the first three sentences exceed 50 characters, truncate them and add an ellipsis at the end. Otherwise, if the text is more than three sentences, put an ellipsis on the fourth line.
+
+Suppose the text starts with the following lines:
+
+> ARI was designed for military use in 1967. It was for real-time 
+> monitoring of readability on typewriters. It offered an automated 
+> counting method. It has similar results to other frameworks.
+> This validates its accuracy.
+
+Then the program should print:
+
+```text
+The text is:
+ARI was designed for military use in 1967.
+It was for real-time monitoring of readability on ...
+It offered an automated counting method.
+...
+```
+
+## Text Statistics
+
+The program should calculate and print the number of sentences, words 
+and characters in the text. The sentences can end with a full stop, 
```

Comment:
Does it make sense to go from small items to large? Characters, words, sentences.

Reply:
The google request did not give a clear result for "count characters, words, sentences."

> Counting words, letters, sentences, characters
https://kalkulatori.lv/en/word-count

The listing order is not important to me.

Reply:
I would suggest going from small to large here.

Reply:
fixed